packages/localization/src/DateUtils.js

```diff
@@ -25,4 +25,10 @@ DateUtils.diffInDays = (higherDate: Date, lowerDate: Date) => {
   return Math.ceil(diffMs / (1000 * 3600 * 24));
 };
 
+DateUtils.isSameDay = (firstDate: Date, secondDate: Date): boolean => {
+  const localizedFirstDate = firstDate.toLocaleDateString();
+  const localizedSecondDate = secondDate.toLocaleDateString();
+  return localizedFirstDate === localizedSecondDate;
```

Comment:
Why don't you just compare it? I don't see a point in comparing localized strings here.

Reply:
Why do you need `toLocaleDateString` for it?

Reply:
The way I understand it is that getDate() or getMonth() will just extract naively from the UTCZ timestamp. So for example:
2018-12-23T23:00:00.000Z is 2018-12-24T01:00:00.000+02:00

Reply:
Why don't you try it? 🙂

```
> new Date('2018-12-23T23:00:00.000Z').toLocaleDateString('cs')
'2018-12-24'
```

Reply:
I am confused by your comment. That's what my `isSameDay` function does, comparing the output of `toLocaleDateString`.

Reply:
Aha, I didn't notice it's called `isSameDay` and not `isSameDate`...